packages/spatial/src/input/functions/ClientInputHooks.test.tsx

```diff
@@ -53,16 +53,16 @@ import { InputState } from '../state/InputState'
 import ClientInputHooks from './ClientInputHooks'
 
 const createMockHTMLCanvasElement = (ev: MockEventListener) => {
-  return {
-    addEventListener: ev.addEventListener,
-    removeEventListener: ev.removeEventListener,
-    getDrawingBufferSize: () => 0,
-    getContext: () => {},
-    parentElement: {
+  return new (class {
```

Comment:
with https://github.com/ir-engine/ir-engine/pull/854 we can retire this custom mock function and use a standardized mock

Reply:
The problem with this is that the standard event listener API does not allow you to check if the event exists in the event listener.